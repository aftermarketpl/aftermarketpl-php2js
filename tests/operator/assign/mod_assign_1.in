$a = 4;
$b = 2;
$a %= $b;
return $a;
