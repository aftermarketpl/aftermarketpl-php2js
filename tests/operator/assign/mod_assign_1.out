var a, b;
a = 4;
b = 2;
a %= b;
return a;
